### ISSUE TITLE
Update data of "Indian Institute of Information Technology" Section

### ISSUE THREAD

**Issue by mohit355** [GDSC-IIIT-Kalyani/placement-portal#33]
current data
![image](https://user-images.githubusercontent.com/54148372/135893268-59571194-d2ca-409c-aac5-73863ff4ee3f.png)

New data need to update
![image](https://user-images.githubusercontent.com/54148372/135893205-adf6567c-f30b-4227-ada3-0d2700838f67.png)

**Comment by unnati-bhatia** [comment]
Assign please

**Comment by mohit355** [comment]
@unnati-bhatia  go ahead

**Comment by unnati-bhatia** [comment]
Sir, I have a doubt. Since I have already forked this repo earlier while contributing to some other issue. Should I delete previous fork and do it again or otherwise how?

**Comment by mohit355** [comment]
@unnati-bhatia  no, just fetch and merge using fetch stream option available in your forked repo
![image](https://user-images.githubusercontent.com/54148372/135897197-187cf970-5736-442f-b572-5197974e37ea.png)
